src/Tickets/Emails/Admin/Emails_Tab.php

```diff
@@ -172,4 +197,46 @@ public function get_fields(): array {
 		 */
 		return apply_filters( 'tec_tickets_emails_settings_fields', $fields );
 	}
+
+	/**
+	 * Check if currently editing email.
+	 * 
+	 * @since TBD
+	 *
+	 * @param Email_Abstract $email
+	 * 
+	 * @return boolean
+	 */
+	public function is_editing_email( $email = null ) {
+		if ( ! $this->is_on_tab() ) {
+			return false;
+		}
+
+		// Get `tc-edit` query string from URL.
+		$editing_email  = tribe_get_request_var( static::$edit_query );
```

Comment:
For consistency I'd incline to use directly the `section` parameter (so that we can have this Settings > tab > section logic), because there's plans to update settings in the future with this format. I've added  a helper function for this here: https://github.com/the-events-calendar/event-tickets/pull/2550/files#diff-b864484972892032b7a96254575015028df2c5156fd149628898c69c2b99e626R129

Reply:
Also, for the per email settings I initially thought to have it something like this https://github.com/the-events-calendar/event-tickets/pull/2550/files#diff-c97bfb6de59c73509e252cc9795314ed13c6b61585f3b735b84c66ca9a8a93ffR183

Reply:
So something like `section=edit_email` and then maybe an `id=####` in the URL?

Regarding the settings, I coded it to where it's similar to how the sections on the `Payments` tab work. It shows the general settings unless you're on a particular `section`.